app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml

```diff
@@ -75,7 +75,7 @@ $canApplyMsrp = $helper->isShowBeforeOrderConfirm($product) && $helper->isMinima
                 <span class="pricing msrp">
                     <span class="msrp notice"><?= $block->escapeHtml(__('See price before order confirmation.')) ?></span>
                     <?php $helpLinkId = 'cart-msrp-help-' . $_item->getId(); ?>
-                    <a href="#" class="action help map"
+                    <a href="javascript:void(0);" class="action help map"
```

Comment:
Hi @edenduong. I am not sure that `javascript:void(0);` is the best solution in this case. 
As for me, we should change `<a>` element to `<button>` for a semantic reason and ad.d correct styles for these buttons.

Reply:
@VladimirZaets : Thank you for your feedback. I see so many source code in core now is <a> element:

![image](https://user-images.githubusercontent.com/8234209/64543896-3d409d80-d350-11e9-8100-c230b8269849.png)

I prefer the <a> href in this situation to <button> :) Because in the future, maybe we add the link to the href, and the user can easily click it to redirect to other page.

Reply:
Hi @edenduong. Yes, we have a lot of legacy code, but it doesn't mean that we need to introduce a new code that looks like that.  

The reason that someday "in the future", nobody knows when, but we will add a link is very unstable reason. In this case, we can add href attribute to thousands of another elements. 

I understand your reason but really, change button element to <a> with is not so big deal, and we can do it when we will need it.